web/proxy.js

```diff
@@ -24,13 +24,29 @@ console.log(`Creating proxy with host: ${host}`);
  * environment that has no local API.
  */
 const server = http.createServer((request, response) => {
+    let hostname = host;
+    let requestPath = request.url;
+
+    // regex not declared globally to avoid internal regex pointer reset for each request
+    // We only match for staging related root since default host is prod
```

Comment:
Please update this comment to also explain why we are looking for staging.*api in the request URL. I believe we are doing this so that we only replace the apiRoot on requests to our api, rather than requests to an external source, as with chat attachments. You could provide an example of two request urls and say something like this.

When we make an api request to the proxy the url will look something like this, and we want to replace "staging-api" with the staging api root as defined by the .env config file.
http://localhost:8080/staging-api?command=OpenReport

However when we make a request to view a chat attachment or another external resource it will look like this, and we don't want to replace any part of the URL.
http://localhost:8080/chat-attachments/46233577.....